oaknational/Oak-Web-Application
# Added year sorting function to sort grouped units by year/order

## Description
Added a year sorting function to sort grouped units by year/order. This is because units of a lower years should always sort before units of a greater year, regardless of the order. 

## Issue(s)

Fixes `CUR-969`

## How to test

1. Go to https://deploy-preview-3014--oak-web-application.netlify.thenational.academy
2. Check for any unit order regressions in the visualiser
